src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -446,7 +448,7 @@ public static SnapshotRequestModel createSnapshotRequestByRequestId(
                         new SnapshotRequestIdModel().snapshotRequestId(snapshotAccessRequestId))));
   }
 
-  public static SnapshotAccessRequestModel createAccessRequest() {
+  public static SnapshotAccessRequestModel createAccessRequestModelApproved() {
```

Comment:
Name-wise, could should this match the pattern of `createSnapshotAccessRequestModel`?

Reply:
yes good idea